### PR TITLE
improvement: fix for writing-guidelines #666

### LINKED ISSUE #666: 中文前端優化
幾個看一眼想到的問題

Screenshot 2020-02-25 at 16 01 01

* 行文格式統一：目前有些地方中英文之間有空格，有些沒空。
* 字型選擇雜亂。(#664)
* 右下背景的圖憑空被卡掉很讓人疑惑。（沒 inspect 不知道是圖本來就只有這樣還是區塊的問題）

### PR DESCRIPTION
**Pull request by Sirius207**
Fix #666
- Replace some () to （）
- Replace AoE descriptions

### CODE REVIEW

**Comment by Sirius207** [comment]
@Lee-W

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Lee-W) [inline .vscode/settings.json]: Please remove this file. I suggest you add `.vscode/` to your global `.gitignore` (i.e. `~/.gitignore`)

Comment (Sirius207) [reply .vscode/settings.json]: OK, my bad.
I have removed it.

Comment (Lee-W) [reply .vscode/settings.json]: It's ok. Thanks!

File: src/templates/pycontw-2020/contents/en/speaking/cfp.html [hunk 2]
```diff
@@ -18,18 +18,18 @@ <h1>Call for Proposals </h1>
 <p>The Call for Proposals is now open. PyCon 2020 Taiwan is accepting talks and tutorials!</p>
 
 
-<h2>Important Dates(<a href="https://en.wikipedia.org/wiki/Anywhere_on_Earth">We use AoE timezone.</a>)</h2>
+<h2>Important Dates</h2>
 
 <ul>
 	<li>Talks &amp; Tutorial CfP begins: 1 Mar </li>
-	<li>Talks &amp; Tutorial CfP ends: 26 Apr（23:59）</li>
+	<li>Talks &amp; Tutorial CfP ends: 26 Apr 23:59:59 <a href="https://www.timeanddate.com/worldclock/converter.html?iso=20200427T115900&p1=tz_aoe&p2=33&p3=64&p4=136&p5=44&p6=224&p7=235">AoE</a></li>
```
Comment (uranusjr) [inline src/templates/pycontw-2020/contents/en/speaking/cfp.html]: I’d suggest to simply put

https://www.timeanddate.com/worldclock/converter.html?iso=20200427T115900&p1=tz_aoe&p2=241&p3=1440

The user can add cities if they wish to.

File: src/templates/pycontw-2020/contents/en/speaking/cfp.html [hunk 3]
```diff
@@ -18,18 +18,18 @@ <h1>Call for Proposals </h1>
...
   <li>Announcement of acceptance: In the middle of June (tentative)</li>
   <li>Dates of the Conference：5 Sep - 6 Sep</li>
 </ul>
 
 <p>All talks will be recorded, edited then uploaded to <a href="http://pyvideo.org" target="_blank" rel="noopener">pyvideo.org</a> by the PyCon Taiwan committee, unless the speaker requests otherwise. See more info about the <a href="{{ speaking_recording_url }}">recording release</a>.</p>
 
-<p>All speakers are required to buy their own conference tickets（<a href="http://pyfound.blogspot.com/2017/10/psfs-october-board-meeting.html" target="_blank" rel="noopener">Everyone Contributes Policy</a>）</p>
+<p>All speakers are required to buy their own conference tickets (<a href="http://pyfound.blogspot.com/2017/10/psfs-october-board-meeting.html" target="_blank" rel="noopener">Everyone Contributes Policy</a>)</p>
```
Comment (uranusjr) [inline src/templates/pycontw-2020/contents/en/speaking/cfp.html]: Suggest to use HTTPS here.

File: src/templates/pycontw-2020/contents/zh/speaking/cfp.html [hunk 4]
```diff
@@ -15,14 +15,14 @@
...
+<p>PyCon Taiwan 2020 正式開始徵稿，接受包括議程（talk）與課程（tutorial）的稿件！</p>
 
 
-<h2>重要日期 (採用<a href="https://en.wikipedia.org/wiki/Anywhere_on_Earth">AoE</a>時區)</h2>
+<h2>重要日期</h2>
 
 <ul>
 	<li>議程、課程 開放徵稿：3 月 1 日</li>
-	<li>議程、課程 投稿截止：4 月 26 日（23:59）</li>
+	<li>議程、課程 投稿截止：4 月 26 日（23:59:59 <a href="https://www.timeanddate.com/worldclock/converter.html?iso=20200427T115900&p1=tz_aoe&p2=33&p3=64&p4=136&p5=44&p6=224&p7=235">AoE</a>）</li>
```
Comment (uranusjr) [inline src/templates/pycontw-2020/contents/zh/speaking/cfp.html]: Same here

File: src/templates/pycontw-2020/contents/en/speaking/cfp.html (hunk 2 above)
Comment (Sirius207) [reply src/templates/pycontw-2020/contents/en/speaking/cfp.html]: Sure~

File: src/templates/pycontw-2020/contents/en/speaking/cfp.html (hunk 3 above)
Comment (Sirius207) [reply src/templates/pycontw-2020/contents/en/speaking/cfp.html]: OK, fixed it.

File: src/templates/pycontw-2020/contents/zh/speaking/cfp.html (hunk 4 above)
Comment (Sirius207) [reply src/templates/pycontw-2020/contents/zh/speaking/cfp.html]: OK, fixed it.

**Pull request merged**